pkg/addons/kubectl.go

```diff
@@ -47,6 +47,7 @@ func kubectlCommand(cc *config.ClusterConfig, files []string, enable bool) *exec
 	return exec.Command("sudo", args...)
 }
 
-func kubectlBinaryPath(version string) string {
+// KubectlBinaryPath returns the path to kubectl on the node
+func KubectlBinaryPath(version string) string {
```

Comment:
nit: consider moving this to another package for future proofing cyclic dependcy. maybe kapi package?

Reply:
good idea, done!